thanks.html

```diff
@@ -10,6 +97,99 @@
 		</form>
 		<div id="thanks-message" class="bg-dkgray-trans text-large3 c10 ct10 cm10 box box-huge margin-bottom-large">
 		{% include_tmpl form.thank_you_text %}
+		
+		{% if page.custom_fields.thank_you_page_show_upsell == 'Y' and not action.orderrecurring %}
+			<script>
+				$(function() {
+					var currency = 'USD';
+					var total = '10';
+					var maxDonation = 500;
+					var currencySymbols = {
+						"AUD": "$",
+						"BRL": "R$",
+						"CAD": "CA$",
+						"NZD": "NZ$",
+						"EUR": "€",
+						"GBP": "£",
+						"USD": "$",
+					}
+					
+					{% if action.order.currency %}
+					currency = '{{action.order.currency}}';
+					{% endif %}
+					{% if action.order.total %}
+					total = '{{action.order.total}}';
+					{% endif %}
+					
+					var getCurrency = function(amount) {
+						return parseInt(amount).toLocaleString('en-US', { style: 'currency', currency: currency, minimumFractionDigits: 0, maximumFractionDigits: 0 });
+					}
+					
+					var makeButton = function(amount) {
+						$('#suggestions').append('<button class="upsell_btn button" data-amt="'+parseInt(amount)+'">'+getCurrency(amount)+'/mo</button>')	
+					}
+					
+					var url = 'https://act.350.org/donate/fa-test_recurring/?source=build-donate-ty&utm_source=build-donate-ty&currency='+currency
+					
+					var setupButtons = function() {
+						if (total) {
+							var lowerCalc = (total / 5) * 4
+
+							if (parseInt(lowerCalc) != parseInt(total) && parseInt(lowerCalc) != 0 && parseInt(lowerCalc) <= maxDonation) {
+								makeButton(lowerCalc)
+							} else if(parseInt(total - 1) != parseInt(total) && parseInt(total - 1) != 0 && parseInt(total - 1) <= maxDonation) {
+								makeButton(total - 1)
+							} else if (parseInt(total) > maxDonation) {
+								makeButton((maxDonation / 5) * 4)
+							}
+							if (total < maxDonation) {
+								makeButton(total)	
+							} else {
+								makeButton(maxDonation)
+							}
+						}
+					}
+					
+					$('body').on('click', '.upsell_btn', function(e){
+						window.location.href = url + '&amount='+$(this).data('amt')
+					});
+					
+					$('body').on('click', '#other_amt_btn', function(e){
+						window.location.href = url + '&amount='+$('#other_amt').val()
+					});
+
+					$('#other_symbol').html(currencySymbols[currency])
+					$('#other_amt').addClass(currency)
+					 
+					// used to test
+					// $('#test_no').val(total)
+					 
+					// $('#test_no').on('input', function(e){
+					// 	total = parseFloat($(this).val())
+					// 	$('#suggestions').empty()
+					//     setupButtons();
+					// });
+					 
+					 setupButtons();
+				});
+			</script>
+
+			<!-- used to test -->

```

Comment:
I am thinking this will be removed after we are done testing, yes?

Reply:
Correct! It should be removed if the function of the upsell is all good, that's just there to quickly check how it works without the need to submit lots of donations.

Reply:
@mrkimani-ndegwa removed in 73e6787